app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb

```diff
@@ -0,0 +1,26 @@
+angular.module('formidable').controller('ModalChangeNameCtrl', function ($scope, $modalInstance, $http, $location, $routeParams, $rootScope) {
+
+  url = "<%= ENV['URL'] %>" || 'http://localhost:3000';
+  
+
+  $scope.changeFormName = function(newName) {
+    $scope.form = {
+      name: newName
+    };
+    $http.put(url + '/forms/' + $routeParams.id ,$scope.form).success(function(response) {
+       
+    }).error(function(response){
+      alert("write something you douche");
+    });
+  }
+
+  $scope.ok = function(newName) {
+    $rootScope.$emit('formName',[newName]);
```

Comment:
well noted, I saw with fer and will keep it like this for now. Fer will teach me the right way to do it later on for my personal grow as a developer :+1: